pkg/client/cluster-essentials/go.mod

```diff
@@ -0,0 +1,100 @@
+module github.com/vmware-tanzu/tanzu-framework/pkg/client/cluster-essentials
+
+go 1.18
```

Comment:
Let's use a newer Golang version.

Reply:
if we use greater than 1.18 version make verify (make module) pipeline will fail with 
**"go: go.mod file indicates go 1.19, but maximum version supported by tidy is 1.18"**